Assets/Resources/MagicItemTemplates.txt

```diff
@@ -1,1359 +1,3070 @@
 [
     {
-        "position": 4,
-        "name": "The Masque of Clavicus Vile",
```

Comment:
This was an intentional edit on my part. The item's full name is the "The Masque of Clavicus Vile". I don't mind it being truncated again to match classic however.

Reply:
I didn't realize that you had done manual edits to the file, I thought it was just automatically created.

As for this particular name, I can see an argument for changing it or not. I think maybe whoever made it that way thought of "Clavicus" as being his "first name". It's the same with "The Mehrunes Razor" (rather than "The Mehrunes Dagon Razor". Although it's "The Mace of Molag Bol" and not "The Mace of Molag".) I just checked the Clavicus Vile quest and he's only referred to as "Clavicus Vile", as one uninterrupted name, there. I think it's like that in the other TES games, too, but I generally try to avoid using the later games to retrofit things to Daggerfall. I would probably just leave it as it is in classic, but if you think it should be "The Masque of Clavicus Vile", I'm fine with that, too.

Reply:
Let's just leave it as-is then. :)